src/chat/summarizers/topics.ts

```diff
@@ -0,0 +1,16 @@
+import { LanguageModelTextPart } from "vscode";
+import { KafkaTopic } from "../../models/topic";
+
+/** Create a bullet-point list of {@link KafkaTopic} names. */
+export function summarizeTopics(topics: KafkaTopic[]): LanguageModelTextPart[] {
```

Comment:
Let's make sure these summarizers just return a (mainly markdown-formatted) single `string`, since we may need to use them outside of the flow of tool-calling (for example, directly adding context to an existing prompt).